docs/receiving.md

```diff
@@ -1,4 +1,65 @@
 # Receiving
 
-### How do I receive?
-To be added later...
+## How do I deposit a transfer?
+
+When someone sends you an transfer, you will receive a notification via email indicating a transfer has been sent to you. The notification includes information about the transfer including:
+
+- Sender's email
+- Transfer amount
+- Timestamp of the transfer
+- The notification also includes a link to deposit your transfer to wallet.
+
+Simply click the link in the notification, login using your Google account or stay anonymously, 
+and deposit the funds into the walllet of your choice.
+
+## Can I deposit a transfer on my mobile device?
+
+Yes. Click the deposit link in the notification email and you will be taken to the receive page where you can select from the 
+following wallets:
+
+> * Drive Wallet (cloud wallet linked to your Google account)
+> * Metamask (Using the last used receiving address)
+> * Ledger Hardware Wallet (Using the last receiving used address)
+
+Note that you can using Metamask and Ledger wallet to receive funds without installing Metamask extension or connecting
+to your hardware wallet device, provided that you choose to use the last-used receiving address.
+
+## Does Chainsfer.io send transfer notifications via email.
+
+Yes. We send a notification email to you once the transfer is ready to be accepted.
+
+## How much does it cost to receive a transfer?
+
+We do not charge anything. You only pay the standard network transaction fee. See [transaction fees](/basics#transaction-fees) for details.
+
+## How long will it take to receive the funds?
+
+Once the notification email arrives, you can immediately accept the transfer. Depending on which wallet you want to use for receiving funds, 
+the transaction times vary and can take anywhere from 15 seconds to 1 hour. See [transaction time](/basics#transaction-time) for details.
+
+## How do I decline an incoming transfer?
+
+You can simply ignore the notification email until the transfer expires (30 days). Upon, we will inform the sender to cancel the transfer.
+
+## What do I do if I don't know the security answer, or I use the wrong answer multiple times?
+
+If you don't know the answer to the security question, contact the sender. 
+If you answer the security question incorrectly more than three times, we will inform the sender to cancel the transfer.
+
+## What if my contact sends me the wrong amount?
+
+If your contact sent you the wrong amount, please reach out to them directly. The sender is able to cancel the transfer.
+
+## Does an incoming transfer expire?
+
+A transfer expires after 30 days if not claimed by the recipient.
+The sender can either cancel the transfer to retrieve the funds or resend another transfer.
+
+## How do I know that the transfer notification email I received is real?
+
+Always be careful prior to clicking on any links within a notification. 
+Only action on email notifications when you were expecting someone to send you money. 
```

Comment:
"Only action on email notifications" sounds weird and wrong...

Reply:
Only respond to an email from us if you are expecting someone to send you money.

Reply:
copied from etransfer faq, no fix

Reply:
It could be right, it just sounds awkward and weird. According to [Cambridge Dictionary](https://dictionary.cambridge.org/dictionary/english/action), when **action** is used next to adv **on**, **action** itself is a noun.
again, it could be right, but it just doesn't sound right. "**act on**" maybe?